src/CheckoutSdk/Sources/ResponseData.cs

```diff
@@ -0,0 +1,16 @@
+using Newtonsoft.Json;
+
+namespace Checkout.Sources
+{
+    /// <summary>
+    /// Defines the response data of a <see cref="SourceResponse"/>.
+    /// </summary>
+    public class ResponseData
```

Comment:
Response data is dynamic and can vary between sources. This should be treated the same way as `AlternativePaymentSourceResponse`